.gitlab-ci.yml

```diff
@@ -240,16 +240,22 @@ examples-clippy-std:
     - job:                         clippy-std
       artifacts:                   false
   script:
+    # Revert as soon as the clippy nightly ice fix lands
```

Comment:
Please also create an issue pointing to this problem so that we do not forget about it.

Reply:
Done! https://github.com/paritytech/ink/issues/721